### PR TITLE
HAWQ-1316. Fix feature test build on CentOS7.3

### PR DESCRIPTION
**Pull request by radarwave**

### CODE REVIEW

File: src/test/feature/Makefile [hunk 1]
```diff
@@ -9,7 +9,7 @@ gmock_lib_path = $(abs_top_srcdir)/depends/thirdparty/googletest/build/googlemoc
 override CXX = g++
 override CXXFLAGS = -Wall -O0 -g -std=c++11
 override CPPFLAGS := -I/usr/include -I/usr/local/include -I/usr/include/libxml2 -I$(abs_top_srcdir)/src/test/feature/ -I$(abs_top_srcdir)/src/test/feature/ManagementTool/ -I$(abs_top_srcdir)/src/test/feature/lib/ -I$(abs_top_srcdir)/src/interfaces/libpq -I$(abs_top_srcdir)/src/interfaces -I$(abs_top_srcdir)/src/include  -I$(gtest_include) -I$(gmock_include)
-override LIBS := $(LIBS) -lgtest -lpq -lxml2 -ltest
+override LIBS := $(LIBS) -lgtest -lpq -lxml2 -ltest -l pthread
```
Comment (radarwave) [inline src/test/feature/Makefile]: Thanks @paul-guo- , yeah, this place is better.

**Comment by paul-guo-** [comment]
+1.

**Comment by liming01** [comment]
+1.

**Comment by xunzhang** [comment]
+1

**Pull request merged**